dependencies.props

```diff
@@ -10,6 +10,8 @@
     </RestoreSources>
   </PropertyGroup>
 
+  <Import Project="eng/Versions.props" />
```

Comment:
Ideally this would be done from a file like `Directory.Build.props`, but most of our tests are not SDK projects so it wouldn't get picked up.